syncAppium.sh

```diff
@@ -17,25 +17,37 @@ do
         simulator=`echo $line | grep simul`
         . ${selenium_home}/getDeviceArgs.sh $udid
 
-        appium=`/usr/bin/curl -s --max-time 30 --connect-timeout 30 http://localhost:${appium_port}/wd/hub/status | grep version`
-#	appium=`ps -ef | grep $appium_home/build/lib/main.js  | grep $udid`
-#	echo appium: $appium
+	appium=`ps -ef | grep $appium_home/build/lib/main.js  | grep $udid`
 
 	if [[ -n "$simulator" ]]; then
 		device=${name}
 	else
 	        device=`cat ${logFile} | grep $udid`
 	fi
-#	echo device: $device
 
-        if [[ -n "$device" && -z "$appium" ]]; then
+	wda=`ps -ef | grep xcodebuild | grep $udid | grep WebDriverAgent`
+        if [[ -n "$appium" && -z "$wda" ]]; then
+                sleep 10
```

Comment:
let's declare variable at the top of this file and comment about: increase this value for slow mac machines (during this pause WDA should be able to start)